docs/en/admins/02_Prerequisites.md

```diff
@@ -7,8 +7,9 @@ You need to verify that your server can run FreshRSS before installing it. If yo
 | Software      | Recommended             | Also Works With         |
 | ------------- | ----------------------- | ----------------------- |
 | Web server    | **Apache 2.4+**         | nginx, lighttpd         |
-| PHP           | **PHP 7.4+**            |                         |
+| PHP           | **PHP 7.4+**            | Soft require Apache 2.4+ (but repair minimal compatibility with Apache 2.2) |
 | PHP modules   | Required: libxml, cURL, JSON, PDO_MySQL, PCRE and ctype.<br />Required (32-bit only): GMP <br />Recommended: Zlib, mbstring, iconv, ZipArchive<br />*For the whole modules list see [Dockerfile](https://github.com/FreshRSS/FreshRSS/blob/edge/Docker/Dockerfile-Alpine#L7-L9)* | |
+| PHP settings  | **memory_limit: 128MB+**, **max_execution_time: 30s+** |                        |
```

Comment:
I am not sure of those values. Should be tested a bit. And we modify the `max_execution_time` a few places in the code

Reply:
I suggest to keep this line for another PR, so that we can merge the part with Apache already

Reply:
I still think we should first merge the Apache info and come back to the PHP requirements in another PR after some actual tests of the needed memory

Reply:
Sure, but it also works with `memory_limit: 1024 MB` for instance, so that would be misleading. Ideally, it should be the minimal memory required, but I have not tested what it is, and it will also vary depending on the feeds and articles (256 MB might not be enough for some extreme use cases), although there has been efforts to keep memory requirements low. And 60s for sure will not work for some big use cases (I have some myself requiring higher values).

Reply:
I will make a few tests on my personal instance (which does not host large use cases, but has many different types of feeds so it should have some good degree of representativeness). 32MB seems to work fine so far

Reply:
I've been running on 128 (the Debian default) with no issues fwiw, but I also think lower should be just fine.

Reply:
For the record, why this should be done carefully / why it matters: this type of information is used for instance to calculate the number of possible concurrent users on a server, thereby also the cost for operating a service, and I think it might soon also be used for CSRD-like green assessment

Reply:
At 16MB, it has worked mostly fine for my test, besides one specific function of the API, which could by the way be good to optimize:

https://github.com/FreshRSS/FreshRSS/blob/4b29e666b06762b4b36438c9370c38bc43121f78/p/api/greader.php#L719

Reply:
https://github.com/FreshRSS/FreshRSS/issues/6136

Reply:
thanks for this discussions.

Let me explain why it is important for me to have this numbers in the documentation:
I use a cheap webspace. The memory_limit is one of the most important prerequisits to understand whether the application can run on my machine or not.
So I want to share here: which minimum memory_limit is needed for a first run and which memory_limit is suggested. Yes, more is better ;)

Reply:
Exactly, @math-GH , so putting wrong numbers is bad. And finding the correct numbers require testing.
Contribution https://github.com/FreshRSS/FreshRSS/pull/6137

Reply:
I am testing something in the lines of: 16MB minimum (after https://github.com/FreshRSS/FreshRSS/pull/6137), more than 64MB recommended. Help welcome.

Reply:
The export/import functions currently do not scale at all, as they put everything in memory https://github.com/FreshRSS/FreshRSS/issues/6139